src/config/config.py

```diff
@@ -89,3 +89,14 @@ def write(self):
         with open(self._path, 'w', encoding='utf-8') as f:
             # pylint: disable-next=no-member
             f.write(self.json())
+
+    def resolve_asset_paths(self) -> dict[str, Path]:
+        """Resolves and returns relative paths for assets defined in config file."""
+        asset_paths = get_field_names(ProjectPaths)
+        resolved_asset_paths = {}
+        for asset_path in asset_paths:
+            resolved_path = resolve_relative_path(
+                self._path, getattr(self.paths, asset_path))
+            resolved_asset_paths[asset_path] = resolved_path
```

Comment:
This should surface an error if one of the asset paths could not be resolved. Otherwise you can't surface that in the CLI.